package.json

```diff
@@ -24,53 +24,60 @@
     "deploy:staging": "SWAGGER_TARGET='s3://mcpportal.cloud.cimpress.io/env-staging/swagger-ui/' BACKUP_SWAGGER_TARGET='s3://mcpportal-backup.cloud.cimpress.io/env-staging/swagger-ui/' ./deploy.sh",
     "deploy:production": "SWAGGER_TARGET='s3://mcpportal.cloud.cimpress.io/env-production/swagger-ui/' BACKUP_SWAGGER_TARGET='s3://mcpportal-backup.cloud.cimpress.io/env-production/swagger-ui/' ./deploy.sh"
   },
+  "greenkeeper": {
+    "ignore": [
+      "jquery",
+      "jquery-migrate",
+      "swagger-client"
+    ]
+  },
   "dependencies": {
     "backbone": "^1.3.3",
-    "bootstrap": "3",
-    "clipboard": "^1.5.16",
+    "bootstrap": "^3",
+    "clipboard": "^1.6.1",
     "handlebars": "^4.0.6",
-    "jquery": "1",
-    "jquery-migrate": "1",
+    "jquery": "^1.12.4",
+    "jquery-migrate": "^1.4.1",
     "marked": "^0.3.6",
-    "swagger-client": "github:swagger-api/swagger-js#v2.1.18",
+    "swagger-client": "^2.1.32",
```

Comment:
note this is safe because swagger-client (swagger-js) fixed the bug that @hsingh23 reported as previously blocking our ability to upgrade this dependency: https://github.com/swagger-api/swagger-js/pull/866